qiita_db/data.py

```diff
@@ -783,16 +772,15 @@ def delete(cls, ppd_id):
             if ppd.status != 'sandbox':
                 raise QiitaDBStatusError(
                     "Illegal operation on non sandboxed preprocessed data")
+            elif ppd.is_submitted_to_ebi:
```

Comment:
As far as I can tell not all these cases are tested, right? Just want to confirm ...